data/FedexDemoData.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<entity-facade-xml type="demo">
+    <mantle.party.Party partyId="Fedex Party" partyTypeEnumId="PtyPerson"/>
+    <moqui.basic.Enumeration description="Fedex REST API" enumId="FedexGateway" enumTypeId="ShippingGatewayType"/>
+    <moqui.basic.Enumeration description="API Token" enumId="FedexApiToken"/>
+    <mantle.shipment.carrier.ShippingGatewayConfig shippingGatewayConfigId="FEDEX_INTEGRATION"
+                                                   shippingGatewayTypeEnumId="FedexGateway" description="Fedex API Integration"
+                                                   getOrderRateServiceName="mantle.fedex.FedexServices.get#OrderShippingRate"
+                                                   validateAddressServiceName="mantle.fedex.FedexServices.validate#PostalAddress"/>
+    <mantle.shipment.carrier.ShippingGatewayOption shippingGatewayConfigId="FEDEX_INTEGRATION" optionEnumId="FedexApiToken"/>
+    <mantle.party.contact.ContactMech contactMechId="Fedex_Contact_Ivan" contactMechTypeEnumId="CmtPostalAddress"/>
+    <moqui.basic.Geo geoId="CAN_ON" geoTypeEnumId="GEOT_STATE" geoName="Ontario" geoCodeAlpha2="ON"/>
```

Comment:
why you are adding geo data here, it should be exists in framework

Reply:
Sure, I have removed this data and modified its usage in services.